docs/hassio_hass.md

```diff
@@ -41,17 +41,17 @@ Update the Hass.io component configuration in your `configuration.yaml` to point
 
 ```yaml
 # configuration.yaml
-hassio:
+frontend:
```

Comment:
This is wrong. Hass.io will also need to be pointed at the polymer repo.

Reply:
Ah got it.  I think I got mixed up.  I was trying to fix a frontend issue not a hassio issue so these docs should still be just hassio and below the only change is doing script/develop instead of script/build_hassio, correct?  I will update.